openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Domains.spec.js

```diff
@@ -58,6 +60,14 @@ describe('Domain page should work properly', () => {
     updateDomainDetails(DOMAIN_1);
   });
 
+  it.skip('Assets Tab should work properly', () => {
+    updateAssets(DOMAIN_1);
+  });
+
+  it.skip('Remove Domain from entity should work properly', () => {
+    removeAssets(DOMAIN_1);
+  });
```

Comment:
Why we are skipping these tests?

Reply:
We are awaiting a PR from @07Himank, where domain remove will be updated.